app/connectors/exchange/ProgressResponse.scala

```diff
@@ -88,6 +91,11 @@ case class SiftProgressResponse(
   siftCompleted: Boolean = false
 )
 
+case class EligibleForJobOfferResponse(
```

Comment:
I would say that all of them should have the ProgressResponse suffix. So EligibleForJobOfferProgressResponse, because we use the suffix Response for other things as well.

Reply:
I'll rename.